src/wifi.rs

```diff
@@ -1355,3 +1358,110 @@ impl WifiWait {
         }
     }
 }
+
+#[cfg(all(feature = "async"))]
+pub struct EspAsyncWifiDriver<'d> {
```

Comment:
Strip the `Esp` prefix please. This stuff should be symmetrically named to `WifiDriver`.